book.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end

```

Comment:
- [ ] The Book class should have a method for associating with the Person class. through Rental.

  ```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
  ```